AdobeStockImageAdminUi/view/adminhtml/web/js/signIn.js

```diff
@@ -18,10 +22,12 @@ define([
             // eslint-disable-next-line max-len
             previewProvider: 'name = adobe_stock_images_listing.adobe_stock_images_listing.adobe_stock_images_columns.preview, ns = adobe_stock_images_listing',
             quotaUrl: 'adobe_stock/license/quota',
+            settingsUrl: 'adminhtml/system_config/edit/section/system',
             modules: {
                 source: '${ $.dataProvider }',
                 preview: '${ $.previewProvider }'
-            }
+            },
+            showPopupOnFail: true
```

Comment:
Please add a `loginClick` method that will be used in `signIn` template and proxy to `login` method but add a popup when logging in using click (instead of `showPopupOnFail ` flag).